packages/react-table/src/components/Table/utils/decorators/selectable.tsx

```diff
@@ -43,7 +43,12 @@ export const selectable: ITransform = (
       : {
           checked: allRowsSelected,
           'aria-label': 'Select all rows'
-        })
+        }),
+    ...(rowData &&
+      rowData.disableCheckbox && {
+        disabled: true,
+        class: 'pf-c-check__input'
```

Comment:
Can you `import styles from '@patternfly/react-styles/css/components/Check/check'` and use `styles.checkInput` instead? This will allow us to more easily stay up to date with Core.

Reply:
Yeah sure!